test/unittest/CMakeLists.txt

```diff
@@ -8,9 +8,24 @@ file(GLOB_RECURSE UNIT_TEST_SOURCE "*.cc")
 add_executable(${PROJECT_NAME}_unit_tests ${UNIT_TEST_SOURCE})
 set_property(TARGET ${PROJECT_NAME}_unit_tests
         PROPERTY RUNTIME_OUTPUT_DIRECTORY ${PRIVATE_RUNTIME_DIR})
+
+# File downloads
+message(STATUS "Downloading http://data.mxnet.io/data/cifar10/cifar10_val.rec...") 
+file(DOWNLOAD "http://data.mxnet.io/data/cifar10/cifar10_val.rec"
```

Comment:
Are we downloading entire cifar10_val.rec. Since this is unit test, I think this will add unnecessary delay to test. Can we have a minimal test file of few bytes?